Server/package.json

```
@@ -4,15 +4,19 @@
     "version": "0.1.0",
     "private": true,
     "scripts": {
-        "dev": "dotenvx run --env-file=.env.public --env-file=.env -- node ./bin/www"
+        "dev": "nodemon ./src/app.js"
```

If you don't include the `dotenvx `command, then the file .env.public will be exlcuded

The entry file of this module is `./bin/www`. Modifying `./src/app.js` is not necessary
```suggestion
        "dev": "dotenvx run --env-file=.env.public --env-file=.env -- nodemon ./bin/www"
```

IMPORTANT: you must have a local .env file and .env.public in order to run this command. Just create those file if you were missing